src/components/callout/callout.e2e.ts

```diff
@@ -0,0 +1,70 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('limel-callout', () => {
+    it('renders a callout with the default "Info" heading and no description', async () => {
+        const page = await newE2EPage();
+        await page.setContent('<limel-callout></limel-callout>');
+        const element = await page.find('limel-callout');
+        expect(element).toHaveClass('callout');
+        const heading = await element.find('.heading');
+        expect(heading.textContent).toEqual('Info');
+        const description = await element.find('.description');
+        expect(description).toBeNull();
+    });
+
+    it('renders a callout with the provided heading', async () => {
+        const page = await newE2EPage();
+        await page.setContent(
+            '<limel-callout heading="Important"></limel-callout>'
+        );
+        const element = await page.find('limel-callout');
+        const heading = await element.find('.heading');
+        expect(heading.textContent).toEqual('Important');
+    });
+
+    it('renders a callout with the provided description', async () => {
+        const page = await newE2EPage();
+        await page.setContent(
+            '<limel-callout description="This is an important callout."></limel-callout>'
+        );
+        const element = await page.find('limel-callout');
+        const description = await element.find('.description');
+        expect(description.textContent).toEqual(
+            'This is an important callout.'
+        );
+    });
+
+    it('does not display description if no description prop provided', async () => {
+        const page = await newE2EPage();
+        await page.setContent('<limel-callout></limel-callout>');
+        const description = await page.find(
+            'limel-callout >>> .main .description'
+        );
+        expect(description).toBeNull();
+    });
+
+    it('renders a callout with the provided type', async () => {
+        const page = await newE2EPage();
+        await page.setContent('<limel-callout type="warning"></limel-callout>');
+        const element = await page.find('limel-callout');
+        expect(element).toHaveClass('warning');
+    });
+
+    it('renders a callout with the correct icon based on type', async () => {
+        const page = await newE2EPage();
+        await page.setContent('<limel-callout type="example"></limel-callout>');
+        const element = await page.find('limel-callout');
+        const icon = await element.find('limel-icon');
```

Comment:
when I run the tests, I get this `console.error`s
<img width="813" alt="image" src="https://user-images.githubusercontent.com/35954987/227922699-eddaee1c-deeb-4649-8dc9-8cc2c042ed90.png">
So I added these 3 files to `/src/icons`. Hope it's OK.

Reply:
the question is if these svgs should have content, or should be just dummy empty svg fiels

Reply:
You should _definitely not_ add any icons from icons8 to this repo, ever, not even in a branch. If that's what you did, they are now public, and need to be removed asap. If you need files to make the tests pass, you can duplicate one of the existing files in `src/icons/` and just rename it. Those files contain a royalty-free image of some yoga-dude.

Reply:
I've overwritten the files you added with the yoga-dude. Since there are references to the pushed commit on this PR, to be really thorough, we'd have to purge those from GitHub's history too, bit that's a massive pita. Hopefully those links stop working after a while anyway…

Reply:
Oh noes… they keep working forever… I just checked on a PR which was merged and had the branch removed in 2018, and a commit that was replaced by a force-push was still accessible 😖

It's a very minor issue with only three icons though, so I don't think we have to worry too much about it. But do use dummies next time 🤗

Reply:
Ah, thanks. Well, even a monkey with basic Adobe Illustrator skills can create the icons that I copied from them here. And they weren't really exactly the same icons as their. But good to know this. Thanks. 

I've now changed their contents to things that I personally created before.